tests/core/test_initialize.py

```diff
@@ -295,7 +295,7 @@ def test_make_install_plan(self):
                         "function": "install_conda_xsh",
                         "kwargs": {
                             "conda_prefix": "/darwin",
-                            "target_path": "/darwin/lib/python2.6/site-packages/xonsh/conda.xsh"
+                            "target_path": "/darwin/lib/python2.6/site-packages/xontrib/conda.xsh"
```

Comment:
wait, what?

Reply:
Yeah, conda will let you stick these files anywhere, so I figured `xontrib/` was the right place for it.

Reply:
I was more curious about the `lib/python2.6` part of it

Reply:
Oh yeah, I think that is a meaningless PREFIX sting.  Conda itself doesn't support 2.6 either